### PR TITLE
Add Symfony 5 support

### PR DESCRIPTION
**Pull request by MarijnKoesen**
Add support for Symfony 5.

Updating php's minimal requirement to 7.2 because that's what symfony 2 requires.

Also updating phpunit to 8.x, we could go to 9.x but that requires php 7.3, and keeping the requirement at 7.2 seems a better trade off.

### CODE REVIEW

File: Provider/ProviderInterface.php [hunk 1]
```diff
@@ -6,29 +6,9 @@
...
-     * @return int
-     */
-    public function getCount();
-
-    /**
-     * @param int $pageNumber
-     * @return SitemapSectionPage
-     */
-    public function getPage($pageNumber);
+    public function getSection(): SitemapSection;
```
Comment (thijsBreker) [inline Provider/ProviderInterface.php:9]: Can we add a newline between every method?

**Review by thijsBreker — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -2,25 +2,22 @@ language: php
 dist: trusty
 
 php:
-  - 7.1
   - 7.2
```
Comment (henriquemoody) [inline .travis.yml:5]: Shall we really keep supporting this version as well?

**Review by henriquemoody — APPROVED** [review]
Since you're adding the type-hints and all, you may as well (maybe on separate pull request) configure a coding standard tool

File: .travis.yml (hunk 2 above)
Comment (MarijnKoesen) [reply .travis.yml:5]: 7.2 is what symfony 5 supports, and currently (this) library does so too, so why would we drop it? (Like i'm not sure what our users are using and don't want to force 7.4 upon them if we don't need it)

File: Provider/ProviderInterface.php (hunk 1 above)
Comment (MarijnKoesen) [reply Provider/ProviderInterface.php:9]: I will implement CS when this is merged and fix everything 👍

File: .travis.yml (hunk 2 above)
Comment (henriquemoody) [reply .travis.yml:5]: The version 7.2 is not actively supported anymore. The version 7.3 won't be actively supported in 4 months.

For me, not supporting older versions means less maintenance but it's also an statement. The last version of PHPUnit, for example, required at least PHP 7.3, and if you can't upgrade your PHP version you're stuck with an older version of PHPUnit.

By keeping the support we retrain ourselves of using new features of PHP, we're stuck to older versions of other packages, and we're supporting people/companies to not upgrade their PHP version.

Although one could argue that it's not out role to push people upgrading their PHP version, I don't think we're also obliged to keep supporting older versions of PHP.

Comment (MarijnKoesen) [reply .travis.yml:5]: Ok, I will drop the support and tag a new major version.

**Pull request merged**